common/deathreasons/wc_plague_deaths.txt

```diff
@@ -0,0 +1,354 @@
+﻿death_plague_sacrifice = {
+	icon = "death_murder.dds"
+	public_knowledge = yes
+}
+
+# death_murder = {
```

Comment:
First, noticed anomaly line changes.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/131698559-427f5b82-5c70-41bb-bf52-d83c657ab6a2.png)

</details>

You copy vanilla files and comment on them. Clear it.

Reply:
Cleaned up.